pkg/util/kubernetes/apiserver/leaderelection/leaderelection_test.go

```diff
@@ -257,89 +232,6 @@ func TestSubscribe(t *testing.T) {
 
 }
 
-func TestGetLeaderIPFollower_ConfigMap(t *testing.T) {
```

Comment:
client-go removed support for ConfigMap leases with https://github.com/kubernetes/client-go/commit/27bbe765353fa01ed516b11bde1da82ae6f6c3bc

Reply:
Thanks for the PR. We still need to support using configmaps for leader election even if the official package doesn't as leases were introduced in the version 1.14 of Kubernetes. We imported the solution with this PR: https://github.com/DataDog/datadog-agent/pull/17365/files#diff-4ea44693afeae7147e707e7c16e5840480b78076d00103b9680bec712c7e995e
I think we can keep the tests.

Reply:
@AliDatadog I think 
3a0141c will do it.  As an aside, is there a plan to deprecate support for k8s <1.14?  At this point it is [5 years old](https://kubernetes.io/blog/2019/03/25/kubernetes-1-14-release-announcement/)...